extensions/extended-blocks/core-embed/facebook.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { addFilter } from '@wordpress/hooks';
+import { registerBlockVariation, unregisterBlockVariation } from '@wordpress/blocks';
+import { Path, SVG } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+const embedFacebookIcon = {
+	foreground: '#3b5998',
+	src: (
+		<SVG viewBox="0 0 24 24">
+			<Path d="M20 3H4c-.6 0-1 .4-1 1v16c0 .5.4 1 1 1h8.6v-7h-2.3v-2.7h2.3v-2c0-2.3 1.4-3.6 3.5-3.6 1 0 1.8.1 2.1.1v2.4h-1.4c-1.1 0-1.3.5-1.3 1.3v1.7h2.7l-.4 2.8h-2.3v7H20c.5 0 1-.4 1-1V4c0-.6-.4-1-1-1z" />
+		</SVG>
+	),
+};
```

Comment:
Copied over from [Gutenberg](https://github.com/WordPress/gutenberg/blob/ed848a0ac49257c1c8162fef09440a36c48a8375/packages/block-library/src/embed/icons.js#L44-L51), in case it's removed there.